Games/baj.2m.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# <bitbar.title>BAJ</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Justin Honold</bitbar.author>
+# <bitbar.author.github>astrostl</bitbar.author.github>
+# <bitbar.desc>Checks for active turns at http://boiteajeux.net every two minutes</bitbar.desc>
+# <bitbar.image></bitbar.image>
+# <bitbar.dependencies></bitbar.dependencies>
+
+# INSTRUCTIONS: set baj_login=LOGIN and baj_password=PASSWORD in ~/.bitbar
+
+. ~/.bitbar
```

Comment:
I'm concerned that this:

 - Doesn't follow convention. All other scripts set variables within the script and need to be edited
 - May break if BitBar gets a feature like https://github.com/matryer/bitbar/issues/88